packages/admin-ui/src/pages/dashboard/components/ChainCard.tsx

```diff
@@ -37,13 +39,16 @@ function ChainCard(chain: ChainData) {
           />
         )
       ) : error ? (
-        <ProgressBar now={100} variant="danger" />
+        <ProgressBar now={100} variant="warning" />
       ) : (
         <ProgressBar now={100} variant="success" />
       )}
 
       <div className="message">
         <RenderMarkdown source={message} noMargin />
+        {error ? (
+          <a href={path.join(apiUrl, dnpName, "logs")}>More info</a>
```

Comment:
You must never use <a> links in a React site. You have to use programatic navigation with `react-router` and relative links

Reply:
done with last commit